### PR TITLE
1517 | Remove target_address argument from i3cRSTDAA function

### PR DESCRIPTION
**Pull request by pons93**
Resolves [1517](https://focusuy.atlassian.net/browse/BMC2-1517).

## How to test
1. Connect an I3C target to the Supernova (it can be the icm42605 for example)
2. Run `python -m unittest tests.test.TestSupernovaController.test_ccc_rstdaa`.

## What to expect
Test pass.

### CODE REVIEW

File: supernovacontroller/sequential/i3c.py [hunk 1]
```diff
@@ -431,7 +431,7 @@ def format_response_payload(command_name, response):
                 return None
             elif command_name == "ccc_direct_rstact":
                 return response["data"] if response["descriptor"] and response["descriptor"]["dataLength"] > 0 else None
-            elif command_name in ["ccc_unicast_setmrl", "ccc_unicast_setmwl", "ccc_broadcast_setmwl", "ccc_broadcast_setmrl"]:
+            elif command_name in ["ccc_unicast_setmrl", "ccc_unicast_setmwl", "ccc_broadcast_setmwl", "ccc_broadcast_setmrl", "ccc_rstdaa"]:
```
Comment (Salonso1602) [inline supernovacontroller/sequential/i3c.py:434]: Is it necessary here? This CCC as far as I understand does not give back any relevant data
The response I got from the SDK `{'id': 9, 'command': 11, 'name': 'I3C TRANSFER', 'header': {'tag': 'RESPONSE_TO_REGULAR_REQUEST', 'result': 'I3C_TRANSFER_SUCCESS', 'hasData': True}, 'descriptor': {'dataLength': 1, 'errors': ['NO_TRANSFER_ERROR']}, 'data': [0]}`  
I see other broadcast CCCs are here, so maybe it is a more general problem to be solved elsewhere

**Review by Salonso1602 — APPROVED** [review]
I have left a minor comment to have in mind, otherwise LGTM

File: supernovacontroller/sequential/i3c.py (hunk 1 above)
Comment (focus-martin) [reply supernovacontroller/sequential/i3c.py:434]: I agree @Salonso1602. The SETXXX CCCs do not return relevant data in the data field. What actually matters is if the transfer was completed successfully or a NACK error happened.

Comment (focus-martin) [reply supernovacontroller/sequential/i3c.py:434]: I see that the `ccc_setnewda` has a better approach.

**Review by focus-martin — APPROVED** [review]
LGTM. But I suscribe to Rafa's opinion about the response handler.

File: supernovacontroller/sequential/i3c.py (hunk 1 above)
Comment (pons93) [reply supernovacontroller/sequential/i3c.py:434]: You are right. I will move `ccc_rstdaa` with `ccc_setnewda` in that `elif` chain.

**Pull request merged**